internal/main_vm.go

```diff
@@ -31,6 +34,27 @@ func Main() {
 	}
 }
 
+// Find the path to package main by looking at the root Caller.
+func findMainPath() string {
+	pc := make([]uintptr, 20)
```

Comment:
20 could be low, maybe 100 for safety?

```suggestion
	pc := make([]uintptr, 100)
```